.travis.yml

```diff
@@ -1 +1,3 @@
-language:java
+language: java
+before_install:
+  - chmod +x mvnw
```

Comment:
good fix for now, but need to change those permissions eventually and delete this line.